compiler/src/dotty/tools/dotc/core/Annotations.scala

```diff
@@ -61,14 +61,15 @@ object Annotations {
     def tree(implicit ctx: Context) = body
   }
 
-  case class LazyBodyAnnotation(bodyExpr: Context => Tree) extends BodyAnnotation {
+  case class LazyBodyAnnotation(private var bodyExpr: Context => Tree) extends BodyAnnotation {
```

Comment:
Oh actually I guess this is intentional: https://github.com/lampepfl/dotty/blob/master/compiler/src/dotty/tools/dotc/typer/Inliner.scala#L193